bundles/org.openhab.binding.loxone/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -206,7 +206,71 @@
 		<state readOnly="true"></state>
 	</channel-type>
 
-	<channel-type id="iRoomV2ActiveModeTypeId">
+	<channel-type id="iRoomModeTypeId">
+		<item-type>Number</item-type>
```

Comment:
Officially channels that have options should be of item-type String: https://www.openhab.org/docs/developer/bindings/thing-xml.html#state-description

Reply:
@Hilbrand Do you know some background why this is advised to be string? I am thinking what should be the value in such case, just the number as a string or the label transformed into some short text form, so the user can use it when sending a command to the channel?

Reply:
I did some research on where this came from and my advice here seems misguided. It is allowed to have a number as type and the documentation is wrong on this. Although I would advice to use meaningful names when numbers don't have any reference for the user. With semi random numbers it's easy for the user to pick the wrong number or have to look up the number in the readme to see what it means. So you can still keep the Number type if you want to.